app/controllers/pages_controller.rb

```diff
@@ -52,6 +52,13 @@ def redirect_path
   end
 
   def help_params
-    params.permit(:message, :email_address, :name, :prison, :job_type)
+    # Unpermitted params when running development and test env
```

Comment:
Not sure we should do this, do we know why the extra params make an appearance in test?

Reply:
In config/environments/development.rb we has this line config.action_controller.action_on_unpermitted_parameters = :raise, which is why it is a problem in dev and test. As soon as I click on the submit button I got the unpermitted parameters issue. This was the quickest way I was able to progress. I will try to find another solution.

Reply:
Its there deliberately so that we don't accidentally submit invalid parameters (by silently ignoring them, which is the production setting). It's set that way in production so if someone tried to attack your server they don't end filling your error logs with random rubbish, however its useful to have it in dev/test to show you that you're doing something wrong.